Sourcery/Generating/Generator.swift

```diff
@@ -25,7 +25,7 @@ private class TypesReflectionBox: NSObject {
     lazy var classes: [Type] = {
         return self.types.filter { type in
             let isNotClass = type is Struct || type is Enum || type is Protocol
-            return !isNotClass
+            return !isNotClass && !type.isExtension
```

Comment:
My reasoning for that is that we don't know what was extended, was it a objective-c class or 3rd party struct. So we can not say for sure that that extension belongs to classes collection. But it will be still accessible through `types.all` collection. And with `kind` property it will be  possible to distinguish different kinds of types if needed. 
Btw maybe we also need to return "extension" instead of "class" in `kind` property, what do you think?